recipe/meta.yaml

```diff
@@ -3,8 +3,8 @@
 # core package & vendored libs use different version scheme than CPP libs, see
 # https://github.com/grpc/grpc/blob/v1.48.1/CMakeLists.txt#L28-L32
 # the difference of 23 is empirical and most recently changed with 1.54
-{% set core_major = version.split(".")[1] | int - 23 %}
-{% set core_version = core_major | string ~ ".0.0" %}
+{% set core_major = version.split(".")[1] | int - 24 %}
+{% set core_version = core_major | string ~ ".1.0" %}
```

Comment:
If you change the value, please change the comment. Also, not that it's a big problem, but just to note that a few rerender changes (ostensibly from the previous commit) snuck into this one.

Reply:
Yes, the commit is not pretty. I will fix that now that it is working. I mainly run out of time yesterday and want to see whether it passes CI.